ivy/functional/frontends/numpy/matrix/methods.py

```diff
@@ -117,3 +118,13 @@ def any(self, axis=None, out=None):
         if ivy.exists(axis):
             return any(self.A, axis=axis, keepdims=True, out=out)
         return any(self.A, axis=axis, out=out)
+
+    def sum(self, axis=None, dtype=None, out=None, keepdims=False, initial=0, where=True):
```

Comment:
the method should only consist of `axis`, `dtype` and `out` argument according to the numpy documentation: https://numpy.org/doc/stable/reference/generated/numpy.matrix.sum.html